config.cfg

```diff
@@ -0,0 +1,36 @@
+# Bot sample configuration file.
+
+# This file contains all common configuration variables for the bot. Note
+# some must be generated with the script bin/generate_tokens.py, allowing
+# to generate OAuth2 tokens to connect to some external APIs.
+
+[discord]
```

Comment:
shouldn't the discord token for the bot be on this section?

Reply:
No, as the discord token is a secret one.  Secret stuff will go in `secrets.cfg` and a sample is available in the PR (`secrets.sample.cfg`)